src/data.table.h

```diff
@@ -236,6 +236,7 @@ bool islocked(SEXP x);
 SEXP islockedR(SEXP x);
 bool need2utf8(SEXP x);
 SEXP coerceUtf8IfNeeded(SEXP x);
+SEXP protectAsIs(SEXP x, bool dblist);
```

Comment:
in C, `protect` evokes `PROTECT`, seems that's unintentional. Would `preserveAsIs` be appropriate?

Reply:
> in C, protect evokes PROTECT

Sorry, I don't get the point.

Reply:
the name protect is quite commonly used in R C, so using preserve might be better here to avoid confusion

Reply:
Oh, I see.